docs(learn): update date info in Svelte deployment (#32411)
mdn/content: files/en-us/learn/tools_and_testing/client-side_javascript_frameworks/svelte_deployment_next/index.md

- Even though Svelte is a relatively young project there are lots of tutorials and [courses](https://www.udemy.com/topic/svelte-framework/?sort=popularity) available on the web, so it's difficult to make a recommendation.
- Nevertheless, [Svelte.js — The Complete Guide](https://www.udemy.com/course/sveltejs-the-complete-guide/) by [Academind](https://academind.com/) is a very popular option with great ratings.
- [The Svelte Handbook](https://www.freecodecamp.org/news/the-svelte-handbook/), by [Flavio Copes](https://flaviocopes.com/), is also a useful reference for learning the main Svelte concepts.

> - If you prefer to read books, there's [Svelte and Sapper in Action](https://www.manning.com/books/svelte-and-sapper-in-action) by [Mark Volkman](https://twitter.com/mark_volkmann), which was published in October 2020, which you can [preview online](https://livebook.manning.com/book/svelte-and-sapper-in-action/welcome) for free.

Reviewer: My mistake, I didn't read it properly, getting rid of the extra "which" here:

```suggestion
- If you prefer to read books, there's [Svelte and Sapper in Action](https://www.manning.com/books/svelte-and-sapper-in-action) by [Mark Volkman](https://twitter.com/mark_volkmann), published in October 2020, that [you can preview online](https://livebook.manning.com/book/svelte-and-sapper-in-action/welcome) for free.
```

> - If you prefer to read books, there's [Svelte and Sapper in Action](https://www.manning.com/books/svelte-and-sapper-in-action) by [Mark Volkman](https://twitter.com/mark_volkmann), which was published in October 2020, but which you can already [preview online](https://livebook.manning.com/book/svelte-and-sapper-in-action/welcome) for free.

Reviewer: I guess we can adjust this slightly, also:

```suggestion
- If you prefer to read books, there's [Svelte and Sapper in Action](https://www.manning.com/books/svelte-and-sapper-in-action) by [Mark Volkman](https://twitter.com/mark_volkmann), which was published in October 2020, which you can [preview online](https://livebook.manning.com/book/svelte-and-sapper-in-action/welcome) for free.
```
